bodhi/server/models/models.py

```diff
@@ -2233,12 +2233,13 @@ def expire(self):
         if self.expired_date is not None:
             return
 
-        koji_session = buildsys.get_session()
-        try:
-            koji_session.untagBuild(self.build.release.override_tag,
-                                    self.build.nvr, strict=True)
-        except Exception, e:
-            log.error('Unable to untag override %s: %s' % (self.build.nvr, e))
+        if datetime.utcnow() >= self.expiration_date:
```

Comment:
Don't we want to return if it is not expired? The if/else that I proposed seems cleaner to me.